doc/source/whatsnew/v1.0.0.rst

```diff
@@ -474,6 +474,7 @@ Other API changes
   Supplying anything else than ``how`` to ``**kwargs`` raised a ``TypeError`` previously (:issue:`29388`)
 - When testing pandas, the new minimum required version of pytest is 5.0.1 (:issue:`29664`)
 - :meth:`Series.str.__iter__` was deprecated and will be removed in future releases (:issue:`28277`).
+- ``ignore_index`` is added in :meth:`DataFrame.sort_values` and :meth:`Series.sort_values` to be able to reset index after sorting (:issue:`30114`)
```

Comment:
reverse the ordering a bit ```:meth:`DataFrame.sort_values` and :meth:`Series.sort_values` ``` have gained the ``ignore_index``......

Reply:
changed and moved!